build.gradle

```diff
@@ -731,6 +728,9 @@ project(':core') {
   println "Building project 'core' with Scala version ${versions.scala}"
 
   apply plugin: 'scala'
+  scaladoc {
+      enabled = false
+  }
```

Comment:
Do we still need this with your latest change? Also, can you please check the indenting generally? It should be two spaces.

Reply:
>Do we still need this with your latest change?

I thought about taking it out, but I left it in as a fail-safe, thinking that if the build file changes somehow enabling scaladoc for all submodules, `core` will only provide the scaladoc jar file with the license and manifest.

I don't have a strong opinion on this, so I can remove if you want me to.

Ack on the indenting

Reply:
ok, let's maybe add a comment explaining it then.

Reply:
ack

Reply:
I tweaked it a bit